spec/src/main/asciidoc/rationale.asciidoc

```diff
@@ -0,0 +1,80 @@
+
+In contrast, synchronous communication couples services together, binding their uptime, failure, and handling of the load to each other. 
+In a chain of synchronous interactions, the entire conversation can only be successful if all parties in the chain are responsive - if they are all running, processing messages successfully, and not overloaded. 
+If just one party has a problem, all effectively exhibit the same problem.
+Therefore, systems of microservices relying on HTTP tend to be _fragile_, and failures limit their availability.
+Indeed, in a microservice-based architecture, temporal coupling results in a fragile system, with resilience and scaling properties that are worse than a monolith, hence, it is essential for microservice based architectures to embrace asynchronous communication as much as possible. 
+
+The role of the MicroProfile Reactive Messaging specification is to deliver a way to build systems of microservices promoting both location transparency and temporal decoupling, enforcing asynchronous communication between the different parts of the system.
+
+=== Reactive Systems
+
+https://www.reactivemanifesto.org/[Reactive Systems] provide an architecture style to deliver _responsive_ systems. 
+By infusing asynchronous messaging passing at the core of the system, applications enforcing the reactive system's characteristics are inherently elastic and resilient.
+
+image::reactive-system.png[Reactive System Characteristics, 400, 400]
+
+Microservices part of a reactive systems interacts using _messages_.
+The location and temporal decoupling, promoted by this interaction mechanism, exhibits numerous benefits such as:
+
+* Better failure management as the temporal decoupling would avoid message-loss and avoid cascading failures.
+* Improved elasticity as under fluctuating load the system can decide to scale up and down some of the microservices.
+* The ability to introduce new feature quickly as every component receives and publishes messages.
+
+The MicroProfile Reactive Messaging specification aims to deliver applications embracing the characteristics of reactive systems.
+
+=== On JMS and Message Driven Beans
+
+Java EE offers JMS and Message Driven Beans for handling asynchronous communication; however, there are some problems with these specifications:
+
+* Both are designed for a technology landscape where messaging was a heavyweight process used on the edge of the system to hand control of a transaction from system to another; consequently, there is a lot of boilerplate associated with using these technologies.
+* It is assumed their design that consistency is handled using distributed transactions. 
+However, modern message brokers, popular in microservice deployments, such as Apache Kafka, Amazon Kinesis and Azure Event Hubs, do not support XA transactions, rather, message acknowledgment is handled using offsets with at least once delivery guarantees.
```

Comment:
can we replace 
'modern'
with 
'many'
please - I do not want to imply that IBM MQ, for example, in some way not modern - there are many application scenarios where it is an absolutely great fit and IBM spend a lot of money on continuing to make it better on an ongoing basis.  :-)

Reply:
Done, but IMHO, MQ is modern...